packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -288,7 +287,7 @@ class BottomSheetCell extends Component {
 							help
 					  );
 			}
-			return isEmpty( help )
+			return ! help
```

Comment:
As you can see above and below, `help` is used as a string, therefore we can safely use it as a string in the `if()`, simplifying the condition.